src/model/user/user.schema.ts

```diff
@@ -0,0 +1,16 @@
+import { Schema, model } from "mongoose";
+import { IUser } from "./user";
+
+const userSchema = new Schema<IUser>({
+
+    firstName: { type: String, required: true },
+    lastName: { type: String, required: true },
+    email: {type: String, required: true},
+    hashedPassword: {type: String, required: true},
+    bio: {type: String, required: true},
+    recipes: [{ key: String, value: Date}]
+});

```

Comment:
Don't forget to add a timestamp property to this Schema so mongo can auto-keep track of `createAt` and `updatedAt` attributes.